airbyte-cdk/python/airbyte_cdk/sources/declarative/schema/json_file_schema_loader.py

```diff
@@ -89,3 +95,38 @@ def extract_resource_and_schema_path(json_schema_path: str) -> (str, str):
             return "", split_path[0]
 
         return split_path[0], "/".join(split_path[1:])
+
+    def __resolve_schema_references(self, raw_schema: dict) -> dict:
```

Comment:
can you add context to the PR description? Understanding the intent will help us review